src/Tools/Common/Commands/ProcessStatus.cs

```diff
@@ -64,5 +71,40 @@ private static Process GetProcessById(int processId)
                 return null;
             }
         }
+
+        private static string GetArgs(Process process)
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                try
+                {
+                    using (ManagementObjectSearcher searcher = new ManagementObjectSearcher($"SELECT CommandLine from Win32_Process WHERE ProcessId = {process.Id}"))
+                    {
+                        using (ManagementObjectCollection objectCollection = searcher.Get())
+                        {
+                            return objectCollection.Cast<ManagementBaseObject>().SingleOrDefault()?["CommandLine"]?.ToString().Split("  ")?.Last().Replace("\"", "");
+                        }
+                    }
+                }
+                catch (Exception ex) when (ex is Win32Exception || ex is InvalidOperationException)
+                {
+                    return "[Elevated process - cannot determine command line arguments]";
+                }
+
+            }
+            else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+            {
+                try
+                {
+                    string[] cmdArgs = File.ReadAllText($"/proc/{process.Id}/cmdline")?.Split("\0").Skip(1).ToArray();
```

Comment:
It is possible for applications to write their own value into `cmdline` that doesn't necessarily follow the pattern `/path/to/exe\0--opt\0arg`.  It's not exactly portable, but applications can replace the values in `/proc/self/cmdline` by writing to `argv[0]` or using [`setproctitle`](http://manpages.ubuntu.com/manpages/trusty/man3/setproctitle.3.html).  I think it would be better to _not_ skip and just take all entries.  

I would also complete this line with `?? new string[] {}` so that the null case is handled from the `?.` earlier.

Reply:
By removing the skip, the exe's path would be shown twice if there is one has not altered process title. That may confuse the users. I am planning to split the string on "\0" and see if the first element of the array matches with the Processe's module path, if it does, then don't add that to the return string if it doesn't just return the string that was read in.
Is the extra check needed? I just want to provide a clean output for the users to see.